neural-networks/reidentification/human-reidentification/utils/process.py

```diff
@@ -1,16 +1,16 @@
 import depthai as dai
-from depthai_nodes.ml.messages import ImgDetectionExtended
+from depthai_nodes.message import ImgDetectionExtended, ImgDetectionsExtended
```

Comment:
Messages are importable from top level (same as DAI)
```suggestion
from depthai_nodes import ImgDetectionExtended, ImgDetectionsExtended
```